assets/combined.css

```diff
@@ -680,25 +702,28 @@ figure blockquote {
   padding: 26px 0px;
   padding: 1.625rem 0rem;
 }
+
 @media screen and (min-width: 40em) {
   figure blockquote {
     padding: 62px 0px 31px;
-    padding: 3.444444444444445rem 0rem 1.722222222222222rem;
```

Comment:
Looks like is not possible to increase the number of decimals with node-sass.
But I'm also thinking that this might not be a technical issue but a little problem with the approach of setting those spacing rules.

If you want we can close this PR and you can just add the margin I added by yourself since that's what we are actually trying to fix here.

Meanwhile, I have been thinking to make a port of this project to my framework Concise CSS as an add-on and see if I can improve it, including that issue with the decimals, not by increasing them but by not requiring them at all. I'll get back to this if I find a better solution.